flink-connector-gcp-pubsub/src/main/java/org/apache/flink/connector/gcp/pubsub/common/PubSubConstants.java

```diff
@@ -0,0 +1,11 @@
+package org.apache.flink.connector.gcp.pubsub.common;
+
+import org.apache.flink.annotation.PublicEvolving;
+
+/** Constants for PubSub. */
+@PublicEvolving
+public class PubSubConstants {
```

Comment:
Do we really need a separate class for these constants if they are used only in `PubSubSinkV2Builder` and `PubSubSinkV2BuilderTest` ?

Reply:
It is supposed to be base for extension with other constants, but I agree let's leave that when we actually need it